static/main.css

```diff
@@ -285,6 +285,11 @@ table ul li {
     grid-template-columns: auto;
 }
 
+.half {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
`grid-template-columns: 1fr 1fr;` should maybe be `grid-template-columns: 1fr;` and break to two columns in the 600px [media query](https://github.com/govdirectory/website/blob/f9030f08b925364bec676466f63515d09d17e9e2/static/main.css#L400-L421)?

Reply:
Hmm, I think the media query is beyond me. Do you mind fixing this?